core-services/prow/02_config/_config.yaml

```diff
@@ -2249,6 +2249,7 @@ tide:
     - openshift/tektoncd-triggers
     - openshift/verification-tests
     - openstack-k8s-operators/osp-director-operator
+    - ViaQ/log-file-metric-exporter
```

Comment:
we need one of these entries under the block that has "main" and "master" so we can merge on master branch.  Additionally i think we need something like https://github.com/openshift/release/pull/18761  to exclude the repo from the "bz valid" merge blocker